### PR TITLE
Header with Burger menu

### PR DESCRIPTION
**Pull request by szymonpachucki**

### CODE REVIEW

File: src/components/header/Header.scss
Comment (HaniaNassalska) [inline src/components/header/Header.scss:1]: 'npm install sass' is necessary. Without it, the project won't start ;)
This is not SASS ;) This is not a valid BEM. I would also make separate files for .header and .nav.

File: src/components/header/Header.tsx
Comment (HaniaNassalska) [inline src/components/header/Header.tsx:1]: RWD - there is a problem with the size below 520px. The font is also bad on smaller screens.

**Review by HaniaNassalska — CHANGES_REQUESTED** [review]

**Comment by szymonpachucki** [comment]
@HaniaNassalska @Capoverde @lOdwrotMail

**Comment by szymonpachucki** [comment]
fixed json problem @HaniaNassalska @Capoverde @lOdwrotMail

**Review by HaniaNassalska — CHANGES_REQUESTED** [review]
You're not using NavLinks, paths need fixing. Still an issue with RWD. CSS needs a bit more improvement. All according to our conversation :) We're getting closer - it's already looking nice.

File: src/cart/cart.scss [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+    font-weight: 700;
+    display: none;
+}
+
+.message-active {
+    display: block;
+}
+
+
+@media screen and (min-width: 641px) and (max-width: 1199px) {
```
Comment (pawel-peksa) [inline src/cart/cart.scss:107]: Breakpoint should be set to 650
```suggestion
@media screen and (min-width: 650px) and (max-width: 1199px) {
```

**Review by pawel-peksa — CHANGES_REQUESTED** [review]
Please resolve merge conflicts

**Review by HaniaNassalska — APPROVED** [review]

**Pull request merged**